tests/components/openai_conversation/conftest.py

```diff
@@ -27,5 +29,30 @@ async def mock_init_component(hass, mock_config_entry):
     with patch(
         "openai.Engine.list",
     ):
-        assert await async_setup_component(hass, "openai_conversation", {})
+        assert await async_setup_component(
+            hass,
+            "openai_conversation",
+            {
+                "api_key": "bla",
+            },
+        )
+        await hass.async_block_till_done()
+
+
+@pytest.fixture
+async def setup_complete(hass) -> None:
```

Comment:
This fixture doesn't seem needed. It doesn't do anything more than the fixture above.